llvm/lib/Target/ARM/AsmParser/ARMAsmParser.cpp

```diff
@@ -4053,6 +4073,63 @@ static MCRegister MatchRegisterName(StringRef Name);
 
 /// }
 
+static const DenseSet<StringRef> DataTypes{
```

Comment:
This should be moved inside isDataTypeToken to avoid start-up initialization cost.

Reply:
Done thank you